cmd/track/track_bundle.go

```diff
@@ -170,7 +166,10 @@ func trackBundleCmd(track trackBundleFn, pullImage pullImageFn, pushImage pushIm
 	cmd.Flags().StringVarP(&params.input, "input", "i", params.input, "existing tracking file")
 
 	cmd.Flags().StringSliceVarP(&params.bundles, "bundle", "b", params.bundles,
-		"bundle image reference to track - may be used multiple times (required)")
+		"bundle image reference to track - may be used multiple times (required when --git not provided)")
```

Comment:
Is this right? It's required if neither `--git` nor `--input` are provided. Maybe we should just remove the "required" text from this description.